v2.1/cancel-job.md

```diff
@@ -4,7 +4,7 @@ summary: The CANCEL JOB statement stops long-running jobs.
 toc: false
 ---
 
-<span class="version-tag">New in v1.1:</span> The `CANCEL JOB` [statement](sql-statements.html) lets you stop long-running jobs, which include [`IMPORT`](import.html) jobs and enterprise [`BACKUP`](backup.html) and [`RESTORE`](restore.html) tasks.
+<span class="version-tag">New in v1.1:</span> The `CANCEL JOB` [statement](sql-statements.html) lets you stop long-running jobs, which include schema changes, [`IMPORT`](import.html) jobs, and enterprise [`BACKUP`](backup.html) and [`RESTORE`](restore.html) tasks.
```

Comment:
The "new in v1.1" thing here is really weird. Although cancel came about in 1.1, it's no longer new. However the ability to cancel schema changes is new, while the rest aren't, except for IMPORT, which was added in 2.0. So LGTM I guess, but that's weird.

Reply:
I agree. I remember the Docs team had a discussion about this, but can't remember the decision. Will check with the team and update this doc accordingly.

Reply:
Done :)